common/flatpak-run.c

```diff
@@ -2866,10 +2877,23 @@ flatpak_run_setup_usr_links (FlatpakBwrap *bwrap,
       if (g_file_query_exists (runtime_subdir, NULL))
         {
           g_autofree char *link = g_strconcat ("usr", subdir, NULL);
+          g_autofree char *create = NULL;
+
+          if (sysroot != NULL)
+            create = g_strconcat (sysroot, subdir, NULL);
+          else
+            create = g_strdup (subdir);
+
+          g_debug ("%s -> %s", link, create);
```

Comment:
some leftover spew

Reply:
Only if you have `G_MESSAGES_DEBUG` set, which I thought was the whole point of `g_debug()` - it tells you more about what is going on, but only if you ask for it?

Reply:
Yeah, and we do have debug stuff left in the code for debugging, but generally if we do its something that is useful for debugging when we tell users to run with -v.  I'm not sure this is generally useful for that, and if it is it should be a bit more self-describing what its doing.